src/config/settings.js

```diff
@@ -8,19 +8,51 @@ const fs = require('fs');
 const deepmerge = require('deepmerge');
 
 // Load the settings file
-function load() {
-
+/**
+ * @param{optional bool=false} inBrowser
+ * @returns {{kubernetes: {version: string}}}
+ */
+function load(inBrowser) {
+  if (typeof(inBrowser) == "undefined") {
+    inBrowser = false;
+  }
```

Comment:
We always have a modern enough environment to just have a function sig of...

```js
function load(inBrowser = false) {
```

This will set the default value so there's no need to have the if statement right after the function opening.